app/javascript/components/mentoring/request/MentoringRequestPanel.tsx

```diff
@@ -1,6 +1,10 @@
 import React, { useCallback } from 'react'
 import { SessionProps } from '../Session'
-import { MentorDiscussion, MentoringRequest, Iteration } from '../../types'
+import {
+  MentorSessionDiscussion as Discussion,
+  MentorSessionRequest as Request,
```

Comment:
If you preferred the `Mentor` prefix, I'd be fine with that too. e.g.
```typescript
  MentorSessionDiscussion as MentorDiscussion,
  MentorSessionRequest as MentorRequest,
```

And/or the `Props` suffix too.
```typescript
  MentorSessionDiscussion as DiscussionProps,
  MentorSessionRequest as RequestProps,
```

But I'd like to start standardising everywhere whatever we choose.

Reply:
No preference on this. I can't decide either.